packages/client/src/components/Modals/SearchPanel.vue

```diff
@@ -159,26 +158,43 @@ export default {
   },
   computed: {
     ...mapGetters({
-      fetchEligibilityCodes: 'grants/fetchEligibilityCodes',
+      searchFormFilters: 'grants/searchFormFilters',
+      eligibilityCodes: 'grants/eligibilityCodes',
     }),
   },
   methods: {
     ...mapActions({
       createSavedSearch: 'grants/createSavedSearch',
       updateSavedSearch: 'grants/updateSavedSearch',
+      applyFilters: 'grants/applyFilters',
+      fetchEligibilityCodes: 'grants/fetchEligibilityCodes',
     }),
     setup() {
-      // this.fetchEligibilityCodes();
+      this.fetchEligibilityCodes();
     },
     customLimitText(count) {
       return `+${count}`;
     },
     eligibilityLabel({ label }) {
       return label;
     },
+    apply() {
+      const formDataCopy = { ...this.formData.criteria };
+      this.applyFilters(formDataCopy);
+      this.$emit('filters-applied');
+      this.$refs.searchPanelSideBar.hide();
```

Comment:
This is how I was manually showing/hiding the search panel. If you'd prefer I use `isSearchPanelOpen` LMK!